app/assets/javascripts/pregnancies.coffee

```diff
@@ -1,3 +1,8 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).on "click", "#toggle-call-log", ->
+  $(".old-calls").toggleClass("hidden")
+  html = if $(".old-calls").hasClass("hidden") then "View all calls" else "Limit list"
```

Comment:
Worth noting that this toggle the calls on, but not sure it toggles back? We should file as an issue for now